### DISCUSSION TITLE
SEPHEO: The Meaning Of Success

### DISCUSSION THREAD

**Discussion by King-Mob** [sepheocoop/website#7] [Ideas]
What would it mean for SEPHEO to be a success?

I think the easiest answer to this question is to examine the context that we're in. There are a lot of software developers in the UK and around the world, and a small proportion of them work in coops and charity sectors. SEPHEO would be successful if some of those developers chose to work through our coop to get work, and connect with each other.

But we could take a broader view. Is the version of SEPHEO described above just the latest expression of an ordering of the economy that would happen anyway? That is to say, it's good for SEPHEO to be a vibrant sector of the software development marketplace, but is it good for the marketplace itself? Are we moving things forward, or simply reacting to the conditions that arise in our time?

While devs might like to have a high opinion of how important and skillful our work is, that doesn't mean it'll be that way forever. If you go back 40, 50 years, it was the printers that were riding high as the most rewarded skilled workers of the day. There are still people doing high quality printing work (Calverts Coop for example) but they're not as prominent as they used to be. Things change is basically my point. Maybe technology work is already changing with the AI hype cycle starting to crest?

Historically, there have been lots of associations and collectives of workers with lots of different names e.g. guilds, unions, even companies started as a term just meaning a group of workers. With SEPHEO, we're creating this kind of dynamic flow of workers with a vague idea of pop-up collectives. The question is, how active are we being in creating an economy that provides for people's survival and thriving?

The foundational success of SEPHEO, which we haven't achieved yet, looks to me like hundreds of members with maybe a dozen introductions a month. We would basically have captured the vast majority of the market for freelancers with coops. SEPHEO would be a key player in the supply of software work. But what if we went further than that?

By further, I'm talking about reaching into the demand side of the equation and instantiating things to work on. There's a lot of stuff broken in our society and having reasonable, clear, efficient, predictable software programs and an easy way to access the people who create them, seems like a good start for fixing things.

An example of what I'm gesturing towards might be something like partnering with the Coop Bank to offer loans or grants to coops to improve their internal systems, or transition to actually good open-source software. Or maybe it's something like releasing a new currency or time-banking that links up to lots of other organisations. It's exciting to think of creative ways to go beyond just looking for customers.

Basically I'm looking at the economy as a vast organisation of what people actually spend their time doing. Coops and other organisations have definitely improved that over time, but I worry about being confined to a relatively small corner while the action goes on at large corporations and governments. 

So this is what I'm thinking about with SEPHEO's possible success, kind of a Mondragon level of ambition, maybe even beyond that.

**Comment by wu-lee** [comment]
I like the idea of proactively creating opportunities for work. (Paid work; plenty of opportunities for unpaid work exist.) This is one of the appeals of being entrepreneurially freelance - as opposed to just working as a contractor. I've a pipe dream of supplying teams for solving social and environmental problems which corresponds approximately to the above.

However, there will be challenges, such as keeping the playing field level, and bridging the gap between those who aspire to do and those who can do.  Something to think about.

**Comment by shles** [comment]
I love it! I have a very similar vision of things. And I can propose some specific ideas. This type of space can also empower individuals with solidarity. For example shared resources, like subscriptions to developer services and tools. Even if it is a personal based tool, corporate pricing can be cheaper.

Think about it as Software Consumer Cooperative.

It can lead to financial tools as you mentioned. Loans, insurance and payed time off are things that are hardly available to individual developers, but can be provided inside bigger organisation.
There are examples of that in the art world - [guilded.coop](guilded.coop)
